demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3961 @@
+{
```

Comment:
<p>I do wonder if this breaks the "flow" of the notebook - progressing from toy examples to CORA, then back to toy examples again for heterogeneous?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
That's a fair concern. My thought here was that it helps remind people of the whole pipeline, of actually getting their data off disk. Some options:

1. leave as is, but rewrite the text to be a bit more flow-ful
2. remove the Cora code entirely, focusing on just the synthetic examples, similar to the NetworkX notebook
3. replace this main code with a synthetic example, and move the Cora code to the end, to a section like "Example: loading from CSV files" or something that does all the appropriate `read_csv`s. It can likely be reduced because it wouldn't have to explain all the details.

What do you think?

Reply:
I think it's valuable to have the CORA example - it's the practical use of Pandas, and now we've hidden loading code in datasets it's a big help for people tackling new datasets without delving into our source.

My preference would be Option 3, followed by Option 1 but happy for you to make the call (eg 3. requires more rework).

Reply:
I'll try 3.

Reply:
I think 3 turned out quite well; it makes the notebook much nicer. 👍

Reply:
It flows really well now, thanks for the change! 👍 👍